sql_generators/stable_views/__init__.py

```diff
@@ -481,3 +488,7 @@ def generate(target_project, output_dir, log_level, parallelism, use_cloud_funct
             ),
             one_schema_per_dataset,
         )
+
+
+if __name__ == "__main__":
+    generate()
```

Comment:
I added this to make it easier to run this as a script and attach a debugger.  I forgot to remove it but I think it's fine to keep